docroot/themes/humsci/su_humsci_gin_admin/su_humsci_gin_admin.theme

```diff
@@ -53,3 +53,20 @@ function su_humsci_gin_admin_preprocess(&$variables, $hook) {
   $variables['wrapper_attributes'] = new Attribute();
   $variables['wrapper_attributes']->addClass($classes);
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function su_humsci_gin_admin_preprocess_html(&$variables, $hook) {
+  // Get the current path.
+  $current_path = \Drupal::service('path.current')->getPath();
+  $path_alias = \Drupal::service('path_alias.manager')->getAliasByPath($current_path);
+  // Check if the path matches '/admin/dashboard'.
+  if (preg_match('~^/admin/dashboard(/.*)?$~', $path_alias) || 
+    preg_match('#^/admin/structure/dashboard/[^/]+/preview$#', $path_alias)
+  ) {
```

Comment:
Regexes are not great for performance and here 2 are being executed for each admin page. Let's switch to use the route name instead:

```
$dashboard_routes = ['dashboard', 'entity.dashboard.canonical', 'entity.dashboard.preview'];
$route_name = \Drupal::routeMatch()->getRouteName();
if (in_array($route_name, $dashboard_routes)) {
  // In a dashboard, add class and load library.
}
```

I think that the 3 route names that I added in the example are the only ones you need, but you can confirm that with a `dump()`.